### PR TITLE
Automatically label Azure/TC/Travis PRs

### PR DESCRIPTION
**Pull request by gsnedders**

### CODE REVIEW

**Comment by foolip** [comment]
Ping @jugglinmike

File: lib/metadata/labels.js [hunk 1]
```diff
@@ -45,7 +45,10 @@ var LABEL_TO_PATTERNS = {
         "tools/**"
     ],
     "testdriver.js": ["resources/testdriver*"],
-    "testharness.js": ["resources/testharness*", "resources/test/**"]
+    "testharness.js": ["resources/testharness*", "resources/test/**"],
+    "Azure Pipelines": [".azure-pipelines.yml", "tools/ci/azure/**"],
+    "Taskcluster": [".taskcluster.yml"],
+    "Travis CI": [".travis.yml"]
```
Comment (jugglinmike) [inline lib/metadata/labels.js]: This line is no longer applicable as of https://github.com/web-platform-tests/wpt/pull/16086

Comment (foolip) [reply lib/metadata/labels.js]: I pushed a commit to remove this line.

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
Looks like Travis is going south. https://travis-ci.com/web-platform-tests/wpt-pr-bot/builds/132908010 is queued but hasn't started running. I saw it delayed quite a bit on https://github.com/web-platform-tests/wpt-pr-bot/pull/86 too.

**Comment by foolip** [comment]
I've run `npm run test` locally and it passes, so merging now.

**Comment by foolip** [comment]
@jugglinmike how does deployment work for this repo? I see no CI setup for it.

What I really want to know is when I can remove the old labels and trust the bot won't recreate them.

**Comment by foolip** [comment]
https://github.com/web-platform-tests/wpt/pull/19820 is evidence that it was deployed and works, so I'll remove the old labels now.

**Pull request merged**